buddyboss/buddyboss-platform
# #2264 - Fix: Importing PHBB forum is not working

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2264 .

### How to test the changes in this Pull Request:

1. Go to BuddyBoss > Tools > import Forums
2. Select PHBB in "Select Platform", fill up the rest
3. See all import (Forum, discussion, and reply) statuses

### Proof Screenshots or Video
https://vimeo.com/638645918/7e6dd67c6d

Here is the DB and notes: https://drive.google.com/file/d/1R78kRECqW9NI0lVvb6z8E20CRbW6IZKr/view?usp=sharing

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
